rdev/R/utils.R

```diff
@@ -196,41 +196,60 @@ expand.ev <- function(...) {
 is.numeric.data.frame <- function(x) sapply(x, is.numeric)
 
 
+
+##' Render an \code{mrgsolve} model.
+##' 
+##' @param x model object
+##' @param file file name to use for building; should have \code{.Rmd} extension
+##' @param quiet not used
+##' @param ... not used
```

Comment:
you could pass ... to the rmd render function to add some additional customization